vic/drivers/cesm/src/cesm_interface_c.c

```diff
@@ -34,7 +34,7 @@ all_vars_struct    *all_vars = NULL;
 atmos_data_struct  *atmos = NULL;
 x2l_data_struct    *x2l_vic = NULL;
 l2x_data_struct    *l2x_vic = NULL;
-dmy_struct          dmy;
+dmy_struct          dmy_current;
```

Comment:
Why would this not be a pointer into `dmy`? There doesn't seem to be a good reason to pass this around as a structure rather than a pointer to a structure, since the actual value is stored in the `dmy` array. Unless this is done so that `dmy` is only maintained in the master process and `dmy_current` is broadcast to all processes.